ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -275,11 +280,63 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+        )
+        gui_parser.add_argument(
+            "-H",
+            "--hostname",
+            action="store",
+            help="Set GUI server host address.",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-p",
+            "--port",
+            action="store",
+            type=int,
+            help="Set GUI server host port.",
+            default=None,
+        )
+        return gui_parser
+
+    @staticmethod
+    def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
+        if args.hostname is not None:
+            host = args.hostname
+        else:
+            host = "127.0.0.1"
+
+        if args.port is not None:
+            port = args.port
+        else:
+            port = "8080"
+
+        ofrak = server.OFRAK(server.logging.INFO)
+
+        server.LOGGER.warning(
+            "No disassembler backend specified, so no disassembly will be possible"
+        )
+
+        url = f"http://{host}:{port}"
+        webbrowser.open(url)
```

Comment:
If this doesn't print the URL, consider adding a print statement that the GUI is being served on this URL. If a user closes the tab or doesn't see the window open, this will tell them how to access it.